### PR TITLE
Projectkk2glider/issue 3167 image support

### PR DESCRIPTION
**Pull request by projectkk2glider**
This pull will be used to develop PNG (and other formats support)

Re #3167

### CODE REVIEW

File: radio/src/gui/horus/menu_general_sdmanager.cpp [hunk 1]
```diff
@@ -360,7 +360,17 @@ bool menuGeneralSdManager(evt_t _event)
   if (ext && !strcasecmp(ext, BITMAPS_EXT)) {
     if (lastBitmap != menuVerticalPosition) {
       lastBitmap = menuVerticalPosition;
-      if (bmpLoad(modelBitmap, reusableBuffer.sdmanager.lines[index], MODEL_BITMAP_WIDTH, MODEL_BITMAP_HEIGHT)) {
+      if (imgLoad(modelBitmap, reusableBuffer.sdmanager.lines[index], MODEL_BITMAP_WIDTH, MODEL_BITMAP_HEIGHT)) {
```
Comment (bsongis) [inline radio/src/gui/horus/menu_general_sdmanager.cpp]: It seems duplicate code, no?

File: radio/src/gui/horus/lcd.h [hunk 2]
```diff
@@ -311,6 +311,8 @@ void lcdSetContrast();
 #define lcdOff(...)
 
 const char * bmpLoad(uint8_t * dest, const char * filename, uint16_t width, uint16_t height);
+const char * imgLoad(uint8_t * bmp, const char * filename, uint16_t width, uint16_t height);
```
Comment (bsongis) [inline radio/src/gui/horus/lcd.h]: Parameter is 'dest' in one function and 'bmp' in the other

**Comment by bsongis** [comment]
Good, thanks!

**Comment by projectkk2glider** [comment]
The image loading should now (theoretically) work on the Horus board :wink:

**Comment by bsongis** [comment]
I will have a check now!

**Pull request merged**